### DISCUSSION TITLE
Tailscale support

### DISCUSSION THREAD

**Discussion by WayneManion** [Guilouz/Creality-Helper-Script-Wiki#497] [Features Request]
I don't really know the logistics of it, but it would be very slick to be able to connect to my K1 over Tailscale. I can do it on my printers whose hosts run mainstream Linux distributions like Debian.

**Comment by Guilouz** [comment]
It’s not a Debian here, Creality use a buildroot OS and Tailscale not working with it.
But read my wiki about Guppyflo.

**Reply by jb510** [reply]
GuppyFLO is cool, but adds a web proxy and doesn't* provide direct tailscale access.I am guessing GuppyFLO sticks everything it inside a docker container?  It's seems to create a lot of CPU load and make lots of things sluggish (touch screen, fluid).

It would be preferable to just have tailscale directly.  

* Looking at GuppyFLO's documentation there is a "TCP mode" and I'm unclear if that option was available when installing through Guilouz/Creality-Helper-Script or not, but perhaps that would allow direct access to ports without going through GuppyFLOs nginx proxy.  

Regardless, sure would be nice to have stand alone Tailscale and it does sound like they solved the problems with Tailscale on Buildroot systems https://github.com/tailscale/tailscale/issues/11569

**Comment by WayneManion** [comment]
I knew Creality wasn't using a mainstream Linux base, but I didn't know the correct term for it. Thanks.

GuppyFlo looks pretty cool. The Github says some things about paying for a subscription. Is that for ngrok?

**Comment by Guilouz** [comment]
No you can use tailscale with free account.

**Comment by Tux0Topo** [comment]
Hi! Don´t know if its planned, but i would love to see an update with Tailscale support. Without the need of other apps like GuppyFlo....
Thanks!
